### PR TITLE
Add rejection reasons to UAT branch

### PR DESCRIPTION
**Pull request by RichardBray**
https://jira.digital.csd.fcdo.gov.uk/browse/EIA-156

### CODE REVIEW

File: views/partials/rejectionReasons.ejs [hunk 1]
```diff
@@ -0,0 +1,877 @@
...
+                    <summary class="govuk-details__summary">
+                        <span class="govuk-details__summary-text">
+                            Further information
+                            <span class="govuk-visually-hidden">Document cannot be legalised by e-Apostille
+                                service</span>
+                        </span>
+                    </summary>
+                    <div class="govuk-details__text">
+                        <p class="govuk-body">
+                            Some documents cannot be legalised by the e-Apostille service. You may be able to [use the
```
Comment (simonmoore100) [inline views/partials/rejectionReasons.ejs]: Remove the square bracket

File: views/partials/rejectionReasons.ejs [hunk 2]
```diff
@@ -0,0 +1,877 @@
...
+            <details class="govuk-details govuk-!-margin-bottom-0" data-module="govuk-details">
+                <summary class="govuk-details__summary">
+                    <span class="govuk-details__summary-text">
+                        Further information
+                        <span class="govuk-visually-hidden">There is a problem with the dates</span>
+                    </span>
+                </summary>
+                <div class="govuk-details__text">
+                    <p class="govuk-body">
+                        We are unable to legalise your document(s) because the date on the cover sheet does not match the date the digital
```
Comment (simonmoore100) [inline views/partials/rejectionReasons.ejs]: This sentence doesn't make sense, to me at least

File: views/partials/rejectionReasons.ejs [hunk 3]
```diff
@@ -0,0 +1,877 @@
...
+                        </span>
+                    </summary>
+                    <div class="govuk-details__text">
+                        <p class="govuk-body">
+                            We have been unable to legalise your document(s), because the certification of the solicitor or notary public does not
+                            meet requirements.
+                        </p>
+                        <p class="govuk-body">
+                            When the solicitor or notary public certifies the document(s), they must hold a valid practising certificate in the
+                            country of the UK where the document has been certified.
```
Comment (simonmoore100) [inline views/partials/rejectionReasons.ejs:185]: 'country of the UK'? Is this correct?
I get it's referring to England / Scotland / Wales etc but I had to read it a few times to understand

File: views/partials/rejectionReasons.ejs [hunk 4]
```diff
@@ -0,0 +1,877 @@
...
+                <summary class="govuk-details__summary">
+                    <span class="govuk-details__summary-text">
+                        Further information
+                        <span class="govuk-visually-hidden">Unvalidated certificate
+                        </span>
+                    </span>
+                </summary>
+                <div class="govuk-details__text">
+                        We have been unable to legalise your document because the certificate issued by the Department for International Trade
+                        or Department for Business, Innovation and Skills, contains the wording 'Signature not verified', 'Validity unknown' or
```
Comment (simonmoore100) [inline views/partials/rejectionReasons.ejs:608]: Keep the single quote type consistent

**Review by simonmoore100 — CHANGES_REQUESTED** [review]
The functionality looks to work fine when I run it locally. Just fix up a couple of those typos in the text and we are good to go.

File: views/partials/rejectionReasons.ejs (hunk 3 above)
Comment (RichardBray) [reply views/partials/rejectionReasons.ejs:185]: Yeah, it does sound a bit weird. I had to re-dead it a couple of times but if it's read with the other sentences, (not in isolation), it does make sense.

**Review by simonmoore100 — APPROVED** [review]
LGTM

**Pull request merged**